src/main/resources/openapi/token-1.0.yaml

```diff
@@ -82,13 +87,15 @@ paths:
           $ref: "#/components/responses/trait_400"
         "500":
           $ref: "#/components/responses/trait_500"
-
   /token/refresh:
     parameters:
       - $ref: headers/okapi-tenant-required.yaml
       - $ref: headers/okapi-url-required.yaml
     post:
-      description: Get a new refresh token and a new access token
+      description: |
+        May be called to obtain a new refresh token and a new access token. Returns a signed, expiring
```

Comment:
`May be called to obtain` is very verbose and could be replaced by `Returns`.
`Returns a signed, expiring access token and refresh token.` This can be removed. Expiration is explained in the next sentence, and the client doesn't need information whether the token is signed or encrypted.

Reply:
Reworded